lib/util.js

```diff
@@ -151,7 +151,8 @@ exports.debuglog = function(set) {
     debugEnviron = process.env.NODE_DEBUG || '';
   set = set.toUpperCase();
   if (!debugs[set]) {
-    if (new RegExp(`\\b${set}\\b`, 'i').test(debugEnviron)) {
+    var sets = debugEnviron.split(',').map((s) => s.toUpperCase());
```

Comment:
Oh, I did not notice it is used only in this function. So we can make it the array and use it instead of `sets`, right?